moped-editor/src/components/GridTable/helpers.js

```diff
@@ -136,3 +137,29 @@ export const getDefaultOperator = (filterConfigForField) => {
 
   return isDefaultOperator ? defaultOperator : fallbackOperator;
 };
+
+/**
+ * If filter exists in url, get the values and try to parse them
+ * Used to initialize filter state
+ * @param {Object} searchParams - The URL search parameters
+ * @param {String} advancedSearchFilterParamName
+ * @return {Object}
+ */
+export const useMakeFilterState = (
+  searchParams,
+  advancedSearchFilterParamName
```

Comment:
Small thing that jumped out at me with this `advancedSearchFilterParamName` param is that I kept mixing this up with the specific const that stores the `filters` string used in the url. Maybe naming it something more general could work or maybe it is just my brain 🙃

I also realize that it is only used for that specific search param so maybe it doesn't really matter at all!